staging/src/k8s.io/client-go/scale/client.go

```diff
@@ -196,7 +196,7 @@ func (c *namespacedScaleClient) Update(resource schema.GroupResource, scale *aut
 		Body(scaleUpdateBytes).
 		Do()
 	if err := result.Error(); err != nil {
-		return nil, fmt.Errorf("could not update the scale for %s %s: %v", resource.String(), scale.Name, err)
```

Comment:
I'm guessing you changed this to stop masking errors so that `IsNotFound` and the like would work?  If so, please add a comment.